src/main/resources/com/toptrumps/online/scripts/game.js

```diff
@@ -26,6 +30,7 @@ const Game = (($) => {
         const numberOfOpponents = getNumberOfOpponents();
 
         const request = NetworkHelper.makeRequest("api/game", { numberOfOpponents });
+        Logger.output("Number of opponents", "init", numberOfOpponents);
```

Comment:
Hmm, we should be careful with this name. This Logger cannot be mistaken by the Logger class from server-side.

Reply:
I can add a lot of comments in there. However, if you have doubts, I can get rid of it before we'll be ready to upload.